shared/login/forms/intro.js

```diff
@@ -20,13 +20,19 @@ Intro.propTypes = {
 }
 
 export default connect(
-  state => ({justRevokedSelf: state.login.justRevokedSelf, loaded: state.login.loaded}),
+  state => ({
+    justLoginFromRevokedDevice: state.login.justLoginFromRevokedDevice,
+    justRevokedSelf: state.login.justRevokedSelf,
+    loaded: state.login.loaded
+  }),
   dispatch => ({
     onSignup: () => {
+      dispatch(setLoginFromRevokedDevice(''))
```

Comment:
mmm, kinda gross. likely the action itself should do this if you do login instead of the actual screens. don't have to make this change now but more clear that we have to reset more of these things